data/tasks.txt

```diff
@@ -0,0 +1,10 @@
+T | ? | eat
+E | ✘ | meeting | 20180909 0900
+E | ✘ | meeting | 12/12/2019 900
```

Comment:
Why are the dates and time in different format over here? Also, for event it is a time period as mentioned in the question so might want to add a range instead of just a start time.